package.json

```diff
@@ -43,7 +48,8 @@
     "gulplog": "^1.0.0",
     "jest": "^24.9.0",
     "mustache": "^3.1.0",
+    "pre-commit": "^1.2.2",
```

Comment:
This breaks committing via GitHub Desktop for me. I believe because the precommit hook it installs needs to be ran in a terminal.

Reply:
Well, that is problematic then. 
I'll try to find a workaround.

Reply:
Switched it to using `husky`. Works great from CLI and GitHub Desktop (and appears to be maintained).